examples/utils/oauth.js

```diff
@@ -0,0 +1,21 @@
+// Authenticate with PureCloud
+function authenticate(client, pcEnvironment) {
+    /*
+    * Note: To use this app in your own org, you will need to create your own OAuth2 Client(s)
+    * in your PureCloud org.  After creating the Implicit grant client, map the client id(s) to
+    * the specified region key(s) in the object below, deploy the page, and configure an app to point to that URL.
+    */
+    const pcOAuthClientIds = {'mypurecloud.com': 'implicit-oauth-client-id-here'};
```

Comment:
Nice that now this only lives in one place, especially when we do the injection later in the new build process.